### DISCUSSION TITLE
Load different menus without using the Alias

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#16309] [Q&A]
Is it possibile to load different menus without using the Alias?
The menus have different cultures: english, french, and so on.
I'd like to load them using localization part.

**Comment by Piedone** [comment]
You can use a [MenuWidget](https://github.com/Lombiq/Helpful-Extensions?tab=readme-ov-file#helpful-widgets) and then put it on a culture-specific layer, and do this with all culture-specific menus.

**Reply by DotCat1985** [reply]
> You can use a [MenuWidget](https://github.com/Lombiq/Helpful-Extensions?tab=readme-ov-file#helpful-widgets) and then put it on a culture-specific layer, and do this with all culture-specific menus.

How can load a list of contents using localization part?

**Reply by Piedone** [reply]
You'd need to build the support for that into the menu shape or your theme.
